core/src/main/scala/cats/Functor.scala

```diff
@@ -14,38 +14,42 @@ import simulacrum._
 @typeclass trait Functor[F[_]] extends functor.Invariant[F] { self =>
   def map[A, B](fa: F[A])(f: A => B): F[B]
 
-  /** alias for map */
+  /**
+   *  alias for map 
```

Comment:
I agree with @travisbrown here but also happy to :+1: now. Pareto improvement is what I'm about!